tests/foreman/ui/test_setting.py

```diff
@@ -579,7 +579,31 @@ def test_positive_update_login_page_footer_text(self):
                         self.tab_locator, self.param_name)
                     self.assertEqual(param_value, self.saved_element)
 
-    @stubbed()
+    @tier1
+    def test_positive_remove_login_page_footer_text(self):
+        """Remove parameter "Login_page_footer_text" under General tab
+
+        :id: e31544bf-8d78-4452-9ddb-fd4b76544673
+
+        :expectedresults: Parameter should be removed
+
+        :CaseImportance: Critical
+        """
+        self.tab_locator = tab_locators['settings.tab_general']
+        self.param_name = 'login_text'
+        with Session(self) as session:
+            self.original_value = self.settings.get_saved_value(
+                self.tab_locator, self.param_name)
+            remove_param(
```

Comment:
And what if it was empty already? Imo you should set some value before attempting to remove it, otherwise the test won't really test anything (according to `remove_parameter()` implementation)